examples/adaptiveprecision-blockjacobi/adaptiveprecision-blockjacobi.cpp

```diff
@@ -156,7 +154,7 @@ int main(int argc, char *argv[])
     std::cout << "Final residual norm sqrt(r^T r):\n";
     write(std::cout, lend(res));
     std::cout << "Implicit residual norm squared (r^2):\n";
-    write(std::cout, lend(impl_res));
+    gko::write(std::cout, lend(impl_res));
```

Comment:
I think that is no longer necessary due to ADL, and makes it a bit inconsistent with the previous lines. You could either remove `gko::` or add it to all ADL-found functions (except for lend etc)

Reply:
Of course, seems like I didn't get all of them...